generators/docker-compose/index.js

```diff
@@ -315,7 +318,7 @@ module.exports = DockerComposeGenerator.extend({
             if(this.serviceDiscoveryType === 'consul'){
                 this.template('_consul.yml', 'consul.yml');
             }
-            if(this.serviceDiscoveryType !== 'no'){
+            if(this.serviceDiscoveryType !== false){
```

Comment:
Do you want this changed everywhere or just this instance where serviceDiscoveryType is checked alone?